runtime/shibuya/src/lib.rs

```diff
@@ -1384,11 +1346,25 @@ impl Get<FixedU64> for InitActivePriceGet {
         FixedU64::from_rational(1, 10)
     }
 }
+
+parameter_types! {
+    pub const DappsStakingV2Name: &'static str = "DappsStaking";
+    pub const DappStakingMigrationName: &'static str = "DappStakingMigration";
+}
+
 pub type Migrations = (
     pallet_static_price_provider::InitActivePrice<Runtime, InitActivePriceGet>,
     pallet_dapp_staking_v3::migrations::DappStakingV3TierRewardAsTree<Runtime>,
     pallet_dapp_staking_v3::migrations::DappStakingV3HistoryCleanupMarkerReset<Runtime>,
     pallet_inflation::PalletInflationShibuyaMigration<Runtime, NextEraProvider>,
+    frame_support::migrations::RemovePallet<
```

Comment:
```
    pallet_static_price_provider::InitActivePrice<Runtime, InitActivePriceGet>,
    pallet_dapp_staking_v3::migrations::DappStakingV3TierRewardAsTree<Runtime>,
    pallet_dapp_staking_v3::migrations::DappStakingV3HistoryCleanupMarkerReset<Runtime>,
    pallet_inflation::PalletInflationShibuyaMigration<Runtime, NextEraProvider>,
```
Those have not been applied yet right ?

Reply:
Correct, these are all new.

I've removed the old ones the first time I had to add a new one.